ci/k6-ingestion-test.js

```diff
@@ -1,6 +1,17 @@
 import http from 'k6/http'
-import {check, group, sleep} from 'k6'
+import {check, sleep, fail} from 'k6'
 import { Gauge } from 'k6/metrics'
+import { URL } from 'https://jslib.k6.io/url/1.0.0/index.js';
+
+const POSTHOG_API_ENDPOINT = __ENV.POSTHOG_API_ENDPOINT
+const POSTHOG_EVENT_ENDPOINT = __ENV.POSTHOG_EVENT_ENDPOINT
+
+const captureURL = new URL(`${POSTHOG_EVENT_ENDPOINT}/e/`);
+const apiURL = new URL(`${POSTHOG_API_ENDPOINT}/api/insight/trend/?events=[{"id":"k6s_custom_event","type":"events"}]&refresh=true`);
+
+if (POSTHOG_EVENT_ENDPOINT == null && POSTHOG_API_ENDPOINT == null) {
```

Comment:
Suggestion:

```
if (!POSTHOG_EVENT_ENDPOINT || !POSTHOG_API_ENDPOINT) {
```

If either is missing you want to fail, as with empty strings.